backend/app/processing/vanna_integrator.py

```
@@ -0,0 +1,267 @@
+                #     vn.client.persist()
+                # elif hasattr(vn, "_client"):
+                #     vn._client.persist()
+                # Add more detailed information to the result
+                logger.info("Persisting Chroma database...")
+                try:
+                    # Force persist the ChromaDB collection
+                    if hasattr(vn, "_collection"):
+                        vn._collection.persist()
+                    logger.info("ChromaDB collection persisted successfully")
+                    logger.info("ChromaDB collection persisted successfully")
+                except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)